www/src/components/Menu.tsx

```diff
@@ -144,16 +149,43 @@ interface IManuQueryComponents {
   all: Array<IComponentNavItem>
 }
 
+const foundationLinks = [
+  'Colors', 'Elevation', 'Typography', 'Layout', 'Spacing', 'Icons', 'CSS-Utilities', 'Responsive',
```

Comment:
The hyphen in "CSS-Utilities" gets displayed in the menu, which may not be intentional. I think I'd expect it to say "CSS Utilities" in the menu. Maybe in the `<Link to={`/foundations/${link.toLowerCase().trim()}`}>{link}</Link>` below, you could replace the hyphen with a space?

Reply:
Corrected, thanks